.php_cs

```diff
@@ -17,18 +17,181 @@ $finder = PhpCsFixer\Finder::create()
+        'php_unit_namespaced'                         => ['target' => 'newest'],
+        'php_unit_no_expectation_annotation'          => ['target' => 'newest', 'use_class_const' => true],
+        'php_unit_strict'                             => ['assertions' => ['assertAttributeEquals', 'assertAttributeNotEquals', 'assertEquals', 'assertNotEquals']],
+        'php_unit_test_annotation'                    => ['case' => 'camel', 'style' => 'prefix'],
+        'php_unit_test_class_requires_covers'         => false,
+        'phpdoc_add_missing_param_annotation'         => ['only_untyped' => false],
+        'phpdoc_align'                                => ['tags' => ['param', 'return', 'throws', 'type', 'var']],
+        'phpdoc_annotation_without_dot'               => false,
+        'phpdoc_indent'                               => true,
+        'phpdoc_inline_tag'                           => true,
+        'phpdoc_no_access'                            => true,
+        'phpdoc_no_alias_tag'                         => ['property-read' => 'property', 'property-write' => 'property', 'type' => 'var'],
+        'phpdoc_no_empty_return'                      => false,
+        'phpdoc_no_package'                           => true,
+        'phpdoc_no_useless_inheritdoc'                => true,
+        'phpdoc_order'                                => true,
+        'phpdoc_return_self_reference'                => true,
+        'phpdoc_scalar'                               => true,
+        'phpdoc_separation'                           => true,
+        'phpdoc_single_line_var_spacing'              => true,
+        'phpdoc_summary'                              => false,
+        'phpdoc_to_comment'                           => true,
+        'phpdoc_trim'                                 => true,
+        'phpdoc_types'                                => true,
+        'phpdoc_types_order'                          => ['null_adjustment' => 'always_last', 'sort_algorithm' => 'alpha'],
+        'phpdoc_var_without_name'                     => true,
+        'pow_to_exponentiation'                       => false,
+        'protected_to_private'                        => true,
+        'psr0'                                        => true,
+        'psr4'                                        => true,
+        'random_api_migration'                        => true,
+        'return_type_declaration'                     => ['space_before' => 'none'],
+        'self_accessor'                               => true,
+        'semicolon_after_instruction'                 => true,
+        'short_scalar_cast'                           => true,
+        'silenced_deprecation_error'                  => true,
+        'simplified_null_return'                      => false,
+        'single_blank_line_at_eof'                    => true,
+        'single_blank_line_before_namespace'          => false,
```

Comment:
This goes against what's in Symfony, reason why?

Reply:
fixed